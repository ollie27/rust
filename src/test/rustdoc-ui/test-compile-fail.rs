// compile-flags:--test

/// ```
/// assert!(true)
/// ```
fn f() {}

fn f() {}
