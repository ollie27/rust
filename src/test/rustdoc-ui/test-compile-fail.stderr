error[E0428]: the name `f` is defined multiple times
 --> $DIR/test-compile-fail.rs:8:1
  |
6 | fn f() {}
  | ------ previous definition of the value `f` here
7 | 
8 | fn f() {}
  | ^^^^^^ `f` redefined here
  |
  = note: `f` must be defined only once in the value namespace of this module

error: aborting due to previous error

For more information about this error, try `rustc --explain E0428`.
